src/raven.js

```diff
@@ -1881,7 +1881,36 @@ Raven.prototype = {
     });
   },
 
+  /**
+   * @param {string} opts.url
+   * @param {Object} opts.auth
+   * @param {*} opts.data
+   * @param {Object} opts.options
+   * @callback opts.onSuccess
+   * @callback opts.onError
+   * @private
+   */
   _makeRequest: function(opts) {
+    // NOTE: auth is intentionally sent as part of query string (NOT as custom
+    //       HTTP header) so as to avoid preflight CORS requests
+    var url = opts.url + '?' + urlencode(opts.auth);
+
+    // https://github.com/Modernizr/Modernizr/blob/d5f881a4de0d5fc1af85921ce9c7dc3919c6d335/feature-detects/network/fetch.js
+    if ('fetch' in _window) {
+        return _window.fetch(url, {
+          method: 'POST',
+          body: stringify(opts.data)
+        }).then(function (response) {
+          if (!response.ok) {
+            return opts.onError && opts.onError(new Error('Sentry error code: ' + response.status))
+          }
+
+          opts.onSuccess && opts.onSuccess();
+        })["catch"](function () {
```

Comment:
if you change `})["catch"](function () {` to `}).catch(function () {`
 you get an eslint error